controllers/automation/packagedeployment_controller.go

```diff
@@ -203,20 +225,22 @@ metadata:
 		pkgBuf.Nodes = append(pkgBuf.Nodes, nsNode)
 	}
 
-	// If a ClusterScaleProfile CR exists, and one exists for the
-	// associated cluster, then copy the cluster version to the package
-	if scaleProfileIdx > -1 {
-		clusterScaleProfile, err := r.findClusterScaleProfile(ctx, c)
+	// If a binding resource exists in the package, and one exists
+	// that matches the name of the associated cluster, overwrite the
+	// resource in the package with the cluster resource
+
+	for id, pkgResource := range bindingResources {
+		clusterResource, err := r.findClusterObject(ctx, c, id)
 		if err != nil {
-			r.l.Error(err, "error finding cluster scale profile", "cluster", c)
+			r.l.Error(err, "error finding cluster resource", "cluster", c)
 			return err
 		}
-		if clusterScaleProfile != nil {
+		if clusterResource != nil {
 
 			// convert it to a *RNode
 			var spYamlBuf bytes.Buffer
-			if err := r.s.Encode(clusterScaleProfile, &spYamlBuf); err != nil {
-				r.l.Error(err, "could not write clusterScaleProfile as yaml", "clusterScaleProfile", clusterScaleProfile)
+			if err := r.s.Encode(clusterResource, &spYamlBuf); err != nil {
+				r.l.Error(err, "could not write clusterScaleProfile as yaml", "clusterResource", clusterResource)
```

Comment:
s/clusterScaleProfile/clusterResource/

Actually maybe the actual resource GVK should be used in the description?

Reply:
done!